advice_animal/workflow.py

```diff
@@ -20,8 +20,9 @@ def run(cmd: List[Union[str, Path]], check: bool = True) -> str:
 def files(a: Path) -> Generator[Path, None, None]:
     for root, dirs, files in os.walk(a):
         for f in files:
-            if not f.startswith("."):
-                yield Path(root, f).relative_to(a)
+            if f == ".DS_Store" or (f[0] == "." and f[-4:-1] == ".sw"):
```

Comment:
Should we add `.venv` to prevent it from descending into the virtualenv and finding a bunch of packages there?

Reply:
Nah, this is mainly for selftest today.